projects/tech-zone/new-development-url-notice.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+---
+# New Tech Zone GitHub URL - Notice
+
+**Author:** [Phil Wiffen](https://twitter.com/phil_wiffen)
+
+## We're moving
+
+The Tech Zone contributor location is moving from
+
+<https://github.com/martinzugec-ctx/en-us-tech-zone>
+
+To
+
+<https://github.com/citrixtechzone/en-us-tech-zone>
+
+## Why is this happening?
+
+This will give Citrix a central place for contributors to submit to, that doesn’t have an individual owner.
+
+## What exactly is changing, and what isn't?
+
+Tech Zone is moving from contributors creating content in a Github organisation owned by an individual (martinzugec-ctx), to a Github organisation owned by the Tech Zone team (citrixtechzone).
+
+This will mean that you will need to change where you clone from and push to. But it doesn't change the general contribution process.
+
+## What do I need to do?
```

Comment:
"What if I have an active branch in the old environment"?

Reply:
Good point! I've added some details